pages/index.page.tsx

```diff
@@ -212,26 +212,23 @@ const Home = (props: any) => {
             <div className='lg:w-[650px]  mx-auto my-10 grid grid-cols-1 lg:grid-cols-3 gap-8 justify-items-center '>
               <Link
                 href='/learn/getting-started-step-by-step'
-                className=' flex items-center justify-center rounded border-2 border-white dark:border-none text-white w-[194px] h-[40px] font-semibold bg-primary dark:shadow-2xl'
+                className='flex items-center justify-center rounded border-2 border-white hover:bg-blue-700 transition-all duration-300 ease-in-out text-white w-[194px] h-[40px] font-semibold'
```

Comment:
Here instead of removing `bg-primary` you can add `dark:bg-transparent` since the buttons are a bit darkish blue in light mode which is removed after removing `bg-primary`

Reply:
Here as in dark mode `dark:border-none` there is no border to differentiate the button from the background  adding `dark:bg-transparent` will overwrite and remove the bg color as well making it look like 
![image](https://github.com/json-schema-org/website/assets/96608169/ea70b9d9-1830-41cc-aadf-c32f91773285)
So only adding `bg-primary` is fine